include/swift/AST/DiagnosticsSema.def

```diff
@@ -3296,6 +3296,35 @@ NOTE(availability_conformance_introduced_here, none,
      "conformance introduced here", ())
 
 //------------------------------------------------------------------------------
+// Resilience diagnostics
+//------------------------------------------------------------------------------
+
+ERROR(versioned_attr_with_explicit_accessibility,
+      none, "'@_versioned' attribute can only be applied to internal "
+      "declarations, but %0 is %select{private|fileprivate|internal|public}1",
+      (Identifier, unsigned))
```

Comment:
This should use Accessibility as the argument kind, and I suggest also capitalizing "INTERNAL" and "PUBLIC" (and "OPEN"?) to show that they should never occur.